config/initializers/metadecidim_menu.rb

```diff
@@ -1,10 +1,16 @@
 # frozen_string_literal: true
 
 Decidim.menu :metadecidim_menu do |menu|
-  welcome_path = Decidim::ParticipatoryProcesses::Engine.routes.url_helpers.participatory_process_path("Welcome")
-  menu.add_item :welcome,
-    I18n.t("menu.welcome", scope: "decidim"),
-    welcome_path,
+  menu.add_item :root,
```

Comment:
Getting crowded at the "medium" breakpoint:
![Menu medium](https://user-images.githubusercontent.com/864340/197963540-c46289b1-bd9a-416d-b389-c69dddab2b6f.png)

If you decrease the font-size a bit it would fit:
```scss
.main-nav__link a {
  @include breakpoint(medium) {
    font-size: .9rem;
  }

  @include breakpoint(large) {
    font-size: 1rem;
  }
}
```

While at it, we could also fix the top bar at that breakpoint:
```scss
.topbar__search {
  @include breakpoint(medium) {
    max-width: 120px;
  }

  @include breakpoint(large) {
    max-width: 220px;
  }
}
```

After these changes:
![Top bar medium fixed](https://user-images.githubusercontent.com/864340/197968156-fa868d4d-ab78-455c-9ae5-2a1f6c373c8a.png)

Reply:
Great finding, applied at 71f34494bf90d48d5b586aa6b6bae2f440d55ad7